tests/unit/scala/test_utils_suite.py

```diff
@@ -460,3 +462,26 @@ def test_is_sql_select_statement():
     ]
     for ns in non_select_sqls:
         assert is_sql_select_statement(ns) is False
+
+
+def test_is_snowflake_quoted_id_case_insensitive():
+    assert is_snowflake_quoted_id_case_insensitive('"ABC"')
+    assert is_snowflake_quoted_id_case_insensitive('"AB_C"')
+    assert is_snowflake_quoted_id_case_insensitive('"_AB0C_9"')
+    # negative
+    assert is_snowflake_quoted_id_case_insensitive('"aBC"') is False
```

Comment:
also add a test case without quotes?

Reply:
The input of this method is expected to have quotes. It's DataFrame columns so it's always quoted.